constraints/equalStrength.js

```diff
@@ -0,0 +1,182 @@
+/* Equal Strength Boundaries Constraints
```

Comment:
Max, I apologize for all the comments that should be on this file but which I put on the build/constraints.js file instead...